CRM/Contact/BAO/Group.php

```diff
@@ -384,6 +388,10 @@ public static function create(&$params) {
 
     if (!CRM_Utils_System::isNull($params['parents'])) {
       $params['parents'] = CRM_Utils_Array::convertCheckboxFormatToArray((array) $params['parents']);
```

Comment:
Refactoring note: not a blocker for this PR, but this function really shouldn't be called from the BAO as it's only needed on the quickform.